### DISCUSSION TITLE
关于自建TAPP

### DISCUSSION THREAD

**Discussion by pzjboy0924** [tearust/Chinese-wiki#3] [General]
用户在建Tapp的时候有两种billing模式
第一种，就是Tapp挣不挣钱，你都要给矿工付费；
第二种，是给矿工token，挣钱了按token比例分红

**Comment by kevingzhang** [comment]
第二种模式相当于矿工使用手里的矿机资源作为投资TApp

**Comment by kevingzhang** [comment]
Link已经加入wiki
